iXBRLViewerPlugin/viewer/src/js/viewer.js

```diff
@@ -715,11 +704,8 @@ Viewer.prototype.showDocumentForItemId = function(itemId) {
     this.selectDocument(this._ixNodeMap[itemId].docIndex);
 }
 
-Viewer.prototype.currentDocument = function () {
```

Comment:
@derekgengenbacher-wf is adding a usage of this method in #334.  Can you undo this removal (reimplement in terms of `_currentDocumentIndex`)?

Reply:
Done in 6d693f2.